pom.xml

```diff
@@ -112,9 +112,33 @@
     </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter</artifactId>
+      <artifactId>junit-jupiter-api</artifactId>
       <version>5.8.1</version>
-      <scope>runtime</scope>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
```

Comment:
Is this dependency needed? Is this automatically generated?

Reply:
it is auto-generated, but there are some bugs when generating the organization. That is the only one that I'm changing manually